packages/openneuro-app/src/scripts/dataset/routes/admin-datalad.jsx

```diff
@@ -30,6 +31,8 @@ const AdminDataset = ({ dataset }) => (
     <HeaderRow4>Draft Head</HeaderRow4>
     {dataset.draft.head}
     <DatasetHistory datasetId={dataset.id} />
+    <hr />
+    <DatasetEvents datasetId={dataset.id} />
```

Comment:
We might want to put this above the history? I think this will be more common to reference.

Reply:
moved above history